dvc/_debug.py

```diff
@@ -198,33 +198,83 @@ def debugtools(args: Optional["Namespace"] = None, **kwargs):
 def add_debugging_flags(parser):
     from argparse import SUPPRESS
 
-    parser.add_argument("--cprofile", action="store_true", default=False, help=SUPPRESS)
-    parser.add_argument("--yappi", action="store_true", default=False, help=SUPPRESS)
+    # For detailed info see:
+    # https://github.com/iterative/dvc/wiki/Debugging,-Profiling-and-Benchmarking-DVC
+    dvc_show_debug_options = parser.parse_known_args()[0].verbose
```

Comment:
Nitpicky suggestion, I think it's more clear that we are using `verbose` as a condition for showing the message.

```suggestion
    args, _ = parser.parse_known_args()
    verbose = args.verbose
```